src/black/__init__.py

```diff
@@ -2709,6 +2709,9 @@ def make_comment(content: str) -> str:
 
     if content[0] == "#":
         content = content[1:]
+    NON_BREAKING_SPACE = " "
+    if content and content[0] == NON_BREAKING_SPACE and "type: ignore" not in content:
```

Comment:
Type comments can also be of the form "# type: int". Maybe just check something like `not content.lstrip.startswith("type:")` (assuming that lstrip strips nbsps).

Reply:
There is already an `is_type_comment()` function. It's not quite right though, and I just opened psf/black#2097 about it.

Reply:
Would a regex be ok ? The exact same code than mypy duplicated ?